src/nodejs/jshttpclient.nim

```diff
@@ -46,10 +46,10 @@ func fetchOptionsImpl(request: JsRequest): FetchOptions =
   )
 
 func setHeaders(client: JsHttpClient, request: JsRequest) =
-  client.setRequestHeader([("Integrity".cstring, request.integrity),
-    ("Referer".cstring, request.referer), ("Mode".cstring, request.mode),
-    ("Credentials".cstring, request.credentials), ("Cache".cstring, request.cache),
-    ("Redirect".cstring, request.redirect), ("Referer-Policy".cstring, request.refererPolicy)])
+  client.setRequestHeader([("Integrity".cstring, request.integrity), ("Referer".cstring, request.referer),
```

Comment:
It is `"Referer"` or `"Referrer"` ?, please double check on the whole code...

Reply:
It is "Referer", but in jsfetch "Referrer" is used.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer